packages/edit-post/src/components/header/header-toolbar/style.scss

```diff
@@ -54,7 +54,7 @@
 	}
 
 	// Move toolbar into top Editor Bar.
-	@include break-xlarge {
+	@include break-wide {
```

Comment:
I feel like we keep increasing this breakpoint so it feels like no matter the value used, there will be overflow at some point. I wonder if there's another way to fix this. Maybe a menu that wraps when it hits the limit.

Reply:
The toolbar should collapse into a menu, yes.